ehcache-core/src/main/java/net/sf/ehcache/config/Configuration.java

```diff
@@ -194,7 +197,20 @@ private static void validateOverAllocation(RuntimeCfg config,  Long newValue) {
     private final Map<String, CacheConfiguration> cacheConfigurations = new ConcurrentHashMap<String, CacheConfiguration>();
     private ConfigurationSource configurationSource;
     private boolean dynamicConfig = DEFAULT_DYNAMIC_CONFIG;
+
+    /**
+     * Cache Configuration attribute for size based cache installation
+     *
+     * @deprecated Use {@link #maxBytesLocalOffHeap } instead
```

Comment:
Probably should suggest maxEntriesLocalHeap ?

Reply:
On CacheManager level Heap tier pooling is available only be using maxBytesLocalHeap, maxEntriesLocalHeap is not there. So after maxBytesLocalHeap deprcation, next available pooling option is maxBytesLocalOffHeap which is at the Off-heap tier.